src/index.js

```diff
@@ -1,13 +1,132 @@
+// @flow
+
+import { get } from 'lodash';
+
+export type MongoOptionsType = {
+  collectionName: string,
+  timestamps: {
+    createdAt: string,
+    updatedAt: string,
+  }
+};
+
+export type MongoUserObjectType = {
+  username?: string,
+  profile?: Object,
+  services: {
+    password?: {
+      bcrypt: string,
+    },
+  },
+  emails?: [{
+    address: string,
+    verified: boolean,
+  }],
+};
+
+// TODO Will import from @accounts/account once it's published on npm
+export type UserObjectType = {
+  username: ?string,
+  email: ?string,
+  id: ?string,
+  profile: ?Object,
+  password: ?string,
+};
+
 class Mongo {
```

Comment:
So in terms of pluggable data stores, we should maybe create an interface on the `accounts` package that the Mongo class should implement.
The nice thing is that taking the Mongo methods "as is", is easy.
@TimMikeladze  What are the methods you use at accounts package?